packages/ag-charts-community/src/scene/transformable.ts

```diff
@@ -101,6 +102,10 @@ function MatrixTransform<N extends Node>(Parent: Constructor<N>) {
             return this.toParent(bbox);
         }
 
+        computeBBoxWithoutTransforms() {
+            return super.computeBBox();
+        }
```

Comment:
This is required to get the original dimensions of the bbox before it was transformed, since `super.computeBBox() !== this.fromParent(this.computeBBox())`.